### PR TITLE
add container structure tests

### PR DESCRIPTION
**Pull request by MarckK**

### CODE REVIEW

**Comment by MarckK** [comment]
I've also added in the change to include Makefile in `.gitignore`

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *.tar
 hadolint.json
+Makefile
```
Comment (dduportal) [inline .gitignore:3]:
```suggestion
Makefile

```

Comment (dduportal) [reply .gitignore:3]: I'm being a bit pedantic here, but a newline should terminate each file as stated by POSIX in https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 (a line is always ended by a newline character). There is a clear explanation here: https://thoughtbot.com/blog/no-newline-at-end-of-file . For visual studio, I've this page in my bookmarks to help configuring my editor on new machines, hope ot helps :)

File: Dockerfile [hunk 2]
```diff
@@ -46,6 +47,9 @@ COPY --from=img /usr/bin/newuidmap /usr/bin/newuidmap
...
-ENV USER user
-ENV HOME /home/user
-ENV XDG_RUNTIME_DIR=/run/user/1000
\ No newline at end of file
+ENV USER=user
+ENV HOME=/home/user
+ENV XDG_RUNTIME_DIR=/run/user/1000
+
+CMD ["/bin/bash"]
+WORKDIR "/app"
```
Comment (dduportal) [inline Dockerfile:55]:
```suggestion
WORKDIR "/app"

```

Comment (dduportal) [reply Dockerfile:55]: Same reason as the previous comment

**Review by dduportal — COMMENTED** [review]
This is great work thanks!

Added 2 nit-picking comment which are not blocking :)

**Review by dduportal — APPROVED** [review]
LGTM, great work! Let's merge it and iterate from this

**Pull request merged**